content/hi/docs/tasks/tools/install-kubectl-macos.md

```diff
@@ -0,0 +1,248 @@
+---
+
+title: macOS पर kubectl इंस्टॉल और सेट करें
+content_type: task
+weight: 10
+card:
+  name: tasks
+  weight: 20
+  title: macOS पर kubectl इंस्टॉल करें
+---
+
+## {{% heading "पूर्वाकांक्षित" %}}
```

Comment:
```suggestion
## {{% heading "आवश्यकताए" %}}
```

Reply:
@mittalyashu Shouldn't this be `## {{% heading "prerequisites" %}}`